src/index.js

```diff
@@ -50,7 +50,7 @@ export { ProductTile } from './Tile/Tile';
 export { ProductTileContent } from './Tile/Tile';
 export { ProductTileMedia } from './Tile/Tile';
 export { SearchInput } from './SearchInput/SearchInput';
-export { SideNav, SideNavList, SideNavGroup } from './SideNavigation/SideNavigation';
+export { SideNav, SideNavList, SideNavListItem, SideNavSubList } from './SideNavigation/SideNavigation';
```

Comment:
should `SideNavSubList` still be exported?

Reply:
Fixed